source/dlangbot/app.d

```diff
@@ -465,7 +465,7 @@ void contributorStatsOverall(HTTPServerRequest req, HTTPServerResponse res)
 
 void contributorStatsPerCycle(HTTPServerRequest req, HTTPServerResponse res)
 {
-    contributorStats(req, res, "2022-07-01 23:59:59.000000", "2022-09-30 23:59:59.000000");
+    contributorStats(req, res, "2022-10-01 23:59:59.000000", "2023-01-01 23:59:59.000000");
```

Comment:
The cycle ends at the *end* of January 1? Wouldn't it make more sense to make it end on the start of January 1?

Reply:
You are right. Fixed it. Thanks!